.gitignore

```diff
@@ -13,3 +13,17 @@
 *.diff
 
 xcuserdata
+
+# Created by https://www.toptal.com/developers/gitignore/api/swiftpm
+# Edit at https://www.toptal.com/developers/gitignore?templates=swiftpm
+
+### SwiftPM ###
+Packages
+.build/
+xcuserdata
+DerivedData/
+*.xcodeproj
```

Comment:
This might be what's breaking the project. If I'm not mistaken, the standard Xcode project is going to get nuked by this. 

I'd like peertalk when simply checking out the code, opening Xcode and hitting "Build and Run".

Let's work out a way so that, and Swift PM can live side by side (if possible, if not, let's discuss).